### PR TITLE
Feature presidecms 268 data exports

### PR DESCRIPTION
**Pull request by DominicWatson**
Hi @sebduggan please review.

### CODE REVIEW

File: system/handlers/formcontrols/DataExportFieldPicker.cfc [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	property name="presideObjectService" inject="presideObjectService";
+
+	public string function index( event, rc, prc, args={} ) {
+		var objectName = args.exportObject ?: "";
+
+		if ( !objectName.len() ) {
+			return "";
+		}
+
+		args.defaultValue = dataExportService.getDefaultExportFieldsForObject( objectName ).selectFields.toList();
```
Comment (sebduggan) [inline system/handlers/formcontrols/DataExportFieldPicker.cfc]: Should be down with the other args.

File: system/services/dataExport/DataExportService.cfc [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+			switch( propertyDefinitions[ arguments.fieldName ].type ?: "" ) {
+				case "boolean":
+					return IsBoolean( arguments.value ) ? ( arguments.value ? "true" : "false" ) : "";
+				case "date":
+					if ( !IsDate( arguments.value ) ) {
+						return "";
+					}
+
+					switch( propertyDefinitions[ arguments.fieldName ].dbtype ?: "" ) {
+						case "datetime":
```
Comment (sebduggan) [inline system/services/dataExport/DataExportService.cfc:75]: Could this also be dbtype "timestamp"?

**Review by sebduggan — CHANGES_REQUESTED** [review]
Couple of comments added to the code.

Apart from the code changes we discussed:
- Should time be included in export filename?
- Add docs link to development guides index

**Comment by DominicWatson** [comment]
Closing as review done. Thanks @sebduggan.

**Pull request closed without merging**